sdk/src/client/builder.rs

```diff
@@ -416,3 +331,42 @@ impl ClientBuilder {
         }
     }
 }
+
+/// Struct containing network and PoW related information
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct NetworkInfo {
```

Comment:
Doesn't need a camelCase rename?

Reply:
Ah. It should, but it doesn't break the storage because it's flattened by the client builder 😅